WireMock/wwwroot/scripts/MatcherUI/HeaderMatcherUI.ts

```
@@ -0,0 +1,56 @@
+﻿import HeaderMatcher from "../Matchers/HeaderMatcher.js";
+import MatcherUI from "./MatcherUI.js";
+
+export default class HeaderMatcherUI extends MatcherUI {
+
+    // Render a Matcher row for the HEADER Section
+    // and defining the behaviour of the user inputs
+    static renderHeaderMatcherUI(matcher: HeaderMatcher): HTMLTableRowElement {
+        let matcherRow = document.createElement("tr");
+        matcherRow.setAttribute("id", "matcher-" + matcher.Id);
+        matcherRow.appendChild(this.renderNameCell());
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)